build_tools/preseed_home.py

```diff
@@ -0,0 +1,32 @@
+import os
+import shutil
+import tempfile
+
+temphome = tempfile.mkdtemp()
+os.environ["KOLIBRI_HOME"] = temphome
+
+from kolibri.main import initialize  # noqa E402
+from kolibri.deployment.default.sqlite_db_names import (  # noqa E402
+    ADDITIONAL_SQLITE_DATABASES,
+)
+from django.core.management import call_command  # noqa E402
+
+move_to = os.path.join(os.path.dirname(__file__), "..", "kolibri", "dist", "home")
+shutil.rmtree(move_to)
```

Comment:
First time this is executed this line will trigger a `FileNotFoundError` error . It needs to be in a` try..except `

Reply:
Added `ignore_errors=True`